catalog/app/utils/BucketConfig.ts

```diff
@@ -17,6 +18,9 @@ function useBucketConfigs() {
   // XXX: consider moving this logic to gql resolver
   const empty = cfg.mode === 'MARKETING' || (cfg.alwaysRequiresAuth && !authenticated)
 
+  // TODO: use condition inside GraphQL query
+  const query =
+    cfg.mode === 'PRODUCT' ? BUCKET_CONFIGS_QUERY : BUCKET_CONFIGS_QUERY_NO_COLLABORATORS
   const [{ data }] = urql.useQuery({ query: BUCKET_CONFIGS_QUERY, pause: empty })
```

Comment:
Shouldn't we do this?

```suggestion
  const [{ data }] = urql.useQuery({ query: query, pause: empty })
```